notification/src/server.js

```diff
@@ -0,0 +1,18 @@
+const http = require('http')
+const url = require('url')
+const httpUtils = require('./utils/commons')
+const notificationController = require('./api/notification/notification.controller')
+const config = require('./config/config').load()
+const logger = require('./utils/logger').getLogger(config)
+const routes = {
+  '/': notificationController.handleNotification,
```

Comment:
> How it will work in the reality? When we run our modules, will it have 2 different URLs, one for notification and one for extension?

Since it's was meant to be a serverless app I guess it will be 2 different applications with different containers and deployments.

> What's the point of having this route? 🤔

 I want to add a health endpoint too soon. I want to list all endpoints here.